### DISCUSSION TITLE
GPU usage

### DISCUSSION THREAD

**Discussion by Pableske** [pear-devs/pear-desktop#917] [Q&A]
Hi, is it normal that after installing youtube music my gpu usage goes up to 40% on the desktop? and stays in that range? This happens after installing yt music in my case. is there any way to fix this? because I love this youtube music.thank you and sorry if some words are not understood. i am using translator.

**Comment by Zo-Bro-23** [comment]
What OS are you on? Can you list your device specs (especially GPU)? Thanks!

**Reply by Pableske** [reply]
windows 10 pro - i9-10900k - RTX 3090 I have the pc formatted and it is without the problem. only happens when i install yt music and restart the pc. it's a pity because i really like this youtube music

**Reply by Zo-Bro-23** [reply]
I don't have access to my laptop right now (RTX 3050), but I doubt I experienced the same issue, since I'm likely to have noticed it before. In Task Manager, does it show high GPU percentage for this process, or high GPU usage in general?

One explanation is that you disabled "single instance lock" (disabled by default) and you opened many instances without knowing. Make sure there is only one process running in Task Manager (sort by name).

**Reply by Pableske** [reply]
Thank you for your response.i formatted the pc i don't have yt music installed now because when i install it the gpu percentage goes up to 40 when i'm not using the pc. for example i'm on the desktop with everything closed and it stays between those ranges 35-40 - 45.I remember I looked in the task manager and all that usage was for a process from a file that is in system32.I tried deleting yt music and the problem continued.I also tried deleting the nvidia drivers completely using ddu drivers unistaller.and reinstall the drivers but the problem was still there as well.the problem is when installing yt music where the graphic is set like this, a process consumes a lot of gpu. the process is called csrss.exe. after installing yt music the process called the process is called csrss.exe starts consuming a lot of gpu.

after installing yt music the process called the process is called csrss.exe starts consuming a lot of gpu. sorry if I'm speaking too short or something, I'm using translator and I prefer to write in parts so it translates well.

but it is the csrss.exe process that consumes gpu after installation.

**Reply by Zo-Bro-23** [reply]
[This](https://www.howtogeek.com/321581/what-is-client-server-runtime-process-csrss.exe-and-why-is-it-running-on-my-pc/) article says the following:

![image](https://user-images.githubusercontent.com/62234360/207213951-ad0e0737-e693-4945-a026-20d640bc41fb.png)

If you follow those steps and the file is located in `C:\Windows\System32`, then it's a Windows system process file. If not, it might be malware, but that is unlikely considering you formatted the PC.

From what I understood from your reply, the file _**is**_ located in `System32`, which means that it's a Windows OS process, and nothing to do with malware or third-party applications.

Windows does tend to have bugs that cause it to consume a lot of system resources, which could be the case here.

[This](https://answers.microsoft.com/en-us/windows/forum/all/high-gpu-usage-by-csrssexe-and-dwmexe-even-when/77cb70e4-a724-47c6-9d5c-c25a8b8080c2) thread mentions your same problem, without any apparent solution. [This](https://appuals.com/high-gpu-usage-by-client-server-runtime-and-desktop-window-manager/) article has some potential steps you could try to resolve the issue.

Anyways, the issue isn't caused by YouTube Music, but by Windows itself, so I doubt uninstalling YouTube Music will help. You can continue using YouTube Music! Also, since the issue isn't really pertinent to YouTube Music, could you consider accepting my answer? Thanks!

**Reply by Pableske** [reply]
The strange thing is that it only happens when I install youtube music, since I have formatted 3 times for the same reason. although the use of the gpu may be false.Is it possible to check whether it is a fake or not ?I remember that I installed a program where it showed a real time diagnostic not only of the gpu but of almost all the parts of the pc, where it said the temperatures and all those things.but there I also had the same problem for example if I left the mouse still the gpu usage was at 0 or 5 but when I moved the mouse it went up to 35 - 40. now that I don't have youtube music installed and the pc is formatted there is no such problem.it's very sad for me as I used to use youtube music more on the pc all the time.

**Reply by Zo-Bro-23** [reply]
It might just be that all Chromium-based or Electron apps have the same problem. The Chromium Engine is known to hog system resources. Try installing Discord, Slack, or just Chrome itself, and see if the issue persists.
